src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3149,9 +3170,16 @@ pub struct AlterDefaultPrivilegesStatement<T: AstInfo> {
 impl<T: AstInfo> AstDisplay for AlterDefaultPrivilegesStatement<T> {
     fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
         f.write_str("ALTER DEFAULT PRIVILEGES");
-        if let Some(target_roles) = &self.target_roles {
-            f.write_str(" FOR ROLE ");
-            f.write_node(&display::comma_separated(target_roles));
+        match &self.target_roles {
+            TargetRoleSpecification::Roles(_) => {
+                f.write_str(" FOR ROLE ");
+                f.write_node(&self.target_roles);
+            }
+            TargetRoleSpecification::CurrentRole => {}
+            TargetRoleSpecification::AllRoles => {
+                f.write_str(" FOR ");
+                f.write_node(&self.target_roles);
```

Comment:
Could be tested

Reply:
Is it possible that the coverage report is inaccurate here? I'm almost positive that it's getting tested here: https://github.com/MaterializeInc/materialize/pull/19939/commits/82268626e77ff18f89ee346bd77546a7724da961#diff-93af953bf24ae7fafa2260b763b0f4066e9ae6d1645efb6c6492ebc54925523cR1120-R1126

Reply:
The test definitely starts failing if I modify these lines, so I'm going to assume that it's some inaccuracy with the coverage metrics.

Reply:
Alright, my bad. This seems to have run into one of the coverage bugs.